react/components/EXPERIMENTAL_Table/hooks/useTableMeasures.tsx

```diff
@@ -41,8 +41,10 @@ export function calculateTableHeight(
 }
 
 export function getScrollbarWidth(): number {
-  if (!window || !document || !document.documentElement)
+  const isSSR = typeof document === 'undefined'
```

Comment:
Shouldn't it be like that?
```suggestion
  const isSSR = typeof document === undefined
```

Reply:
nope, it compares as a string [typeof ref](https://developer.mozilla.org/pt-BR/docs/Web/JavaScript/Reference/Operators/typeof)

Reply:
The `typeof` operator always return a string with the type :smile: